src/components/modals/OperationDetails.js

```diff
@@ -226,6 +226,14 @@ const OperationDetails = connect(mapStateToProps)((props: Props) => {
               <OpDetailsTitle>{t('operationDetails.to')}</OpDetailsTitle>
               <DataList lines={recipients} t={t} />
             </Box>
+            {Object.entries(extra).map(([key, value]) => (
+              <Box key={key}>
+                <OpDetailsTitle>
+                  <Trans i18nKey={`operationDetails.extra.${key}`} defaults={key} />
```

Comment:
missed to commit the translation addition?

Reply:
Nope, it defaults to "key" and it's capitalized, so it's find on desktop

Reply:
i'm not sure why making it translatable if it's not in translation then 🤔

Reply:
I guess "Tag" is technical enough to not translate it ? wdyt @dasilvarosa

Reply:
I foresaw the possibility of other extra fields using this, since it's a mapping, and not having a friendly key to display. This case on the desktop it capitalizes the key and it shows as "TAG" so it's ok, if the key had been something like "destinationTag" we would use the translation to rewrite it.

If you don't like it we can remove it, on mobile for example since there's no text-transform I had to use the translation for it to show as "Tag"